symplyphysics/conditions/thermodynamics/critical_point_is_isotherm_stationary_inflection_point.py

```diff
@@ -0,0 +1,31 @@
+from sympy import Eq, Derivative
```

Comment:
Do you have a better name for this condition?

Reply:
The name reflects that for something to be a critical point it needs to be both a stationary and an inflection point of an isotherm. Is there any guidelines you have regarding naming the conditions files?